you-should-use.plugin.zsh

```diff
@@ -135,6 +135,13 @@ function _check_git_aliases() {
         git config --get-regexp "^alias\..+$" | sort | while read key value; do
             key="${key#alias.}"
 
+            # if for some reason, read does not split correctly, we
+            # detect that and manually split the key and value
+            if [[ -z "$value" ]]; then
+                value="${key#* }"
+                key="${key%% *}"
+            fi
```

Comment:
I think you might need to update your base branch? This looks like code that was recently merged into main

Reply:
Sure I'll try to figure out how

Reply:
Fixed